bench_state.go

```diff
@@ -86,6 +86,7 @@ func (s *benchmarkState) printErrors(out output) {
 		total += v
 	}
 	out.Printf("Total errors: %v\n", total)
+	out.Printf("Error rate: %.4f\n", float32(total)/float32(len(s.latencies)))
```

Comment:
That also means that some other stuff is reporting incorrect numbers (e.g. Total Requests in benchmark.go). I'll fix those up as well in this diff.